### PR TITLE
Upgrade request display inside grafana dashboard

### LINKED ISSUE #182: Grafana dashboard missing data
image

### PR DESCRIPTION
**Pull request by MakMuftic**
**Upgrade request display inside grafana (track rate and number of requests in interval for HTTP and WS)**

**Additionally, mute specific data copy error logs inside HTTP tunnel**

### PR Checklist

- [x] I have run linter localy
- [x] I have run unit and integration tests locally
- [x] Rebased to master branch / merged master
- [x] Updated CHANGELOG.md

### Issues

Closes #182

### CODE REVIEW

**Review by mpetrunic — DISMISSED** [review]

**Comment by MakMuftic** [comment]
### New dashboard look
image

### TODO
- there is still possibility that if the number of ws requests is for example **6** dashboard will show **5.5**. This depends on the selected interval and few other things. I will try to resolve this later, as I think this is ok for now.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ jobs:
         with:
           tag_name: ${{ needs.tag.outputs.tag }}
           body: ${{ steps.changelog_reader.outputs.changes }}
-          prerelease: true
+          prerelease: false
```
Comment (MakMuftic) [inline .github/workflows/release.yml:77]: @mpetrunic I am not just sure if I should do this when merging the next release on master or should I change it here?

**Comment by MakMuftic** [comment]
This PR also deals with [this](https://github.com/NodeFactoryIo/vedran-daemon/issues/59) issue on **vedran-daemon**

File: .github/workflows/release.yml (hunk 1 above)
Comment (mpetrunic) [reply .github/workflows/release.yml:77]: here

**Review by mpetrunic — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (MakMuftic) [reply .github/workflows/release.yml:77]: @mpetrunic ^^

Comment (mpetrunic) [reply .github/workflows/release.yml:77]: image

**Pull request merged**